src/drivers/vmount/output.h

```diff
@@ -53,6 +53,13 @@ struct OutputConfig {
 	float gimbal_retracted_mode_value; /**< mixer output value for selecting gimbal retracted mode */
 	float gimbal_normal_mode_value; /**< mixer output value for selecting gimbal normal mode */
 
+	float pitch_range;
```

Comment:
Please store these in radians

Reply:
Is that an optimization or a convention? If it's an optimization, how about storing pitch_scale = 1/(pitch_range * (M_DEG_TO_RAD_F / 2.0f)) instead?

Reply:
Convention: we typically use rad for rotation angles. If you want to further optimize, that's fine with me, just make sure the meaning is clearly documented.

Reply:
Fair enough. Are you happy with the parameters and cached parameter values being in degrees?

Reply:
Yes, since the parameters are user-facing.